samples/excel/85-preview-apis/tetrominos.yaml

```diff
@@ -0,0 +1,783 @@
+
+                function drawConsolePane(session, isPaused, containerElem) {
+                    const consolePaneElem = document.createElement("div");
+                    consolePaneElem.classList.add("consolePane");
+
+                    const previewText = drawPreviewText(session);
+                    const previewElem = drawSessionPreview(session);
+                    const rowsClearedElem = drawRowsCleared(session, isPaused);
+                    const usageElem = drawSessionUsage();
+
+                    consolePaneElem.appendChild(previewText);
+                    consolePaneElem.appendChild(previewElem);
+                    consolePaneElem.appendChild(rowsClearedElem);
+                    consolePaneElem.appendChild(usageElem);
+
+                    containerElem.appendChild(consolePaneElem);
+                }
+
+                function drawRowsCleared(session, isPaused) {
+                    const rowsCleared = session.getRowsCleared();
+                    const rowsClearedElem = document.createElement("div");
+                    rowsClearedElem.classList.add("sessionRowsCleared");
+                    rowsClearedElem.innerHTML = "<p>Rows cleared: " + rowsCleared + "</p>";
+                    if (isPaused) {
+                        rowsClearedElem.innerHTML += "<p>PAUSED</p>";
+                    }
+
+                    if (session.getSessionOver()) {
+                        rowsClearedElem.innerHTML += "<p>SESSION OVER</p>";
+                    }
+
+                    return rowsClearedElem;
+                }
+
+                function drawPreviewText(session) {
+                    const previewTextElem = document.createElement("div");
+                    previewTextElem.classList.add("previewText");
+                    previewTextElem.innerHTML = "<p><i>Next shape</i></p>";
+                    return previewTextElem;
+                }
+
+                function drawSessionPreview(session) {
+                    const piece = session.getNextPiece();
+                    const previewElem = document.createElement("div");
+                    previewElem.classList.add("preview");
+                    drawPreview(piece, 4, 4, previewElem);
+                    return previewElem;
+                }
+
+                function drawSessionUsage() {
+                    const usageElem = document.createElement("div");
+                    usageElem.classList.add("usage");
+                    usageElem.innerHTML =
+                        "<p><b>Keyboard controls</b></p>" +
+                        "<table>" +
+                        "<tr><th>Move shape left</th><td>Left arrow</td></tr>" +
+                        "<tr><th>Move shape right</th><td>Right arrow</td></tr>" +
+                        "<tr><th>Move shape down</th><td>Down arrow</td></tr>" +
+                        "<tr><th>Rotate counterclockwise</th><td>A</td></tr>" +
+                        "<tr><th>Rotate clockwise</th><td>D</td></tr>" +
```

Comment:
@kbrandl There's a MSFT style guide article saying key references should be capitalized. Here's a [public example](https://support.microsoft.com/en-us/help/17073/windows-using-keyboard). My guess is that's because the physical label is always a capital letter.

Reply:
Per our in-person chat just now, let's keep the capital letters :)